util/reggen/gen_dv.py

```diff
@@ -72,6 +72,63 @@ def gen_core_file(outdir: str,
         yaml.dump(core_data, core_file, encoding='utf-8')
 
 
+def get_all_dv_base_names(dv_base_prefix: str) -> Dict[str, List]:
+    ''' Return all DV base names in a dictionary.
+
+    The output format is:
+    {
+      block_1: ["base_reg_block_name", "base_reg_name", "base_reg_field_name", "base_reg_pkg"],
+      block_2: ["base_reg_block_name", "base_reg_name", "base_reg_field_name", "base_reg_pkg"]
+    }
```

Comment:
Sure I will try NamedTuple first. looks like very light weight. Thanks for your suggestion!

Reply:
I think a class would work better in this case because `NamedTuple` is immutable, so updating the value is less convenient than the class. But thanks so much for the suggestion.